packages/caravan-bitcoin/src/fixtures.ts

```diff
@@ -1272,9 +1272,9 @@ const MULTISIGS_BASE = [
+      "cHNidP8BANsBAAAAA1g/HAmf4YziHrug782OS4zxlI5OkrSWO1oguPj+VrNKAAAAAAD/////kJ2FNHdhe4GXlEcem1hxIWXSWs3OOZNV8p4rx004G6IBAAAAAP////8ao1dpKE4oIrZamKxGRyu1xFWDGSfamT2+mtGVnClurwAAAAAA/////wJWVAAAAAAAACIAIDW+dOOdhFLNbt4q7ea+cOIeV5QYQwMuSV2CPrn1Ia6nVx8AAAAAAAAiACCYrErfDscWyy0wWF/I22tBrSZDH1UhNiK0NyPeQYr/QgAAAABPAQSIsh4EC5KBSYAAAAJrLJcFIzT9AzNGT9fXtu2be64l+byr+lTxXgQawilx/AOPzlNRM+qH+YnKCQj36N4PqWXQpmSbcb9cdCROzmTUkBT1fsZdMAAAgAAAAIBkAACAAgAAgE8BBIiyHgS+8hesgAAAAot5AMcDzdD3UrE7epXunLDVoN9/ub/kZ4sHTt09xZoRAwo91ozY06iWIJye7gprjsKHg0WIgs4ac2LYtgs03jCXFAAAAAYwAACAAAAAgGQAAIACAACAAAEA6gIAAAAAAQEao1dpKE4oIrZamKxGRyu1xFWDGSfamT2+mtGVnClurwEAAAAA/v///wIQJwAAAAAAACIAIEl7Amw9NUejDm2ABuOF4DZq9eyitbRV2Hg4dZQeXH+pTHYBAAAAAAAWABRVPpvgr5I4aua0BlJi3Jf97pl5FwJHMEQCIGQHtcH6L9SbkubxgCoO7Z519Vw9seoXnwdwctDG6QMWAiBYI7qNAaa5bbkzSv/Lmx59+0NM/cE4kocBbWtoxcLPDAEhAp3pnC/sb60KkPtad5J3VHe42C+k8hOQOi9n0+LB2ALrmUkJAAEBKxAnAAAAAAAAIgAgSXsCbD01R6MObYAG44XgNmr17KK1tFXYeDh1lB5cf6kBBUdSIQLiG3MYz71IK9u2ZEFCC5AY5bRAv5sM3t1CdibYHzJgWyEDaedPyVQ1W296z5u+xbhhwYaFK3WahfklWOQgoCAgR/RSriIGAuIbcxjPvUgr27ZkQUILkBjltEC/mwze3UJ2JtgfMmBbHAAAAAYwAACAAAAAgGQAAIACAACAAAAAAAAAAAAiBgNp50/JVDVbb3rPm77FuGHBhoUrdZqF+SVY5CCgICBH9Bz1fsZdMAAAgAAAAIBkAACAAgAAgAAAAAAAAAAAAAEA6gIAAAAAAQFYPxwJn+GM4h67oO/NjkuM8ZSOTpK0ljtaILj4/lazSgEAAAAA/v///wLARgEAAAAAABYAFBbYQS13rm1igMYAkacZej+YVGyRECcAAAAAAAAiACBJewJsPTVHow5tgAbjheA2avXsorW0Vdh4OHWUHlx/qQJHMEQCIE+tpv+mH1ePZHrOjI0Ctnj/le0P7Kg7OkdQqXxao93KAiA9xDi1Pnrva9T5FS431svOTmQwaTLMJ/fiBy12SZDLUwEhAgtQASbVbW8Xso6Q8lS0s4wmDs2OlBtlxVe4r2x2cCfdmUkJAAEBKxAnAAAAAAAAIgAgSXsCbD01R6MObYAG44XgNmr17KK1tFXYeDh1lB5cf6kBBUdSIQLiG3MYz71IK9u2ZEFCC5AY5bRAv5sM3t1CdibYHzJgWyEDaedPyVQ1W296z5u+xbhhwYaFK3WahfklWOQgoCAgR/RSriIGAuIbcxjPvUgr27ZkQUILkBjltEC/mwze3UJ2JtgfMmBbHAAAAAYwAACAAAAAgGQAAIACAACAAAAAAAAAAAAiBgNp50/JVDVbb3rPm77FuGHBhoUrdZqF+SVY5CCgICBH9Bz1fsZdMAAAgAAAAIBkAACAAgAAgAAAAAAAAAAAAAEA/QEBAgAAAAABATF7v7GIFKfbH9FeWW+9yLIgt28qveV8BVAkM+JI5dDFAAAAABcWABR+iPsrx0DV7//vi29T1etouDMXtv7///8CECcAAAAAAAAiACBJewJsPTVHow5tgAbjheA2avXsorW0Vdh4OHWUHlx/qdilAQAAAAAAFgAUTd/t/YI+5dyVP0eSqFWyYqhe13wCRzBEAiAvyDR6iAH7vRPbW9JpTNq7LDQots+m7doC/xa8bKy2uAIgVGBNB1EO3lABl97v6dOavid/uXqDeuQOqBW2qjjwMpcBIQJHNIwD9Za1JzBCnqid9LPXWyGf08MMvLZtrgR3ng3m0jZJCQABASsQJwAAAAAAACIAIEl7Amw9NUejDm2ABuOF4DZq9eyitbRV2Hg4dZQeXH+pAQVHUiEC4htzGM+9SCvbtmRBQguQGOW0QL+bDN7dQnYm2B8yYFshA2nnT8lUNVtves+bvsW4YcGGhSt1moX5JVjkIKAgIEf0Uq4iBgLiG3MYz71IK9u2ZEFCC5AY5bRAv5sM3t1CdibYHzJgWxwAAAAGMAAAgAAAAIBkAACAAgAAgAAAAAAAAAAAIgYDaedPyVQ1W296z5u+xbhhwYaFK3WahfklWOQgoCAgR/Qc9X7GXTAAAIAAAACAZAAAgAIAAIAAAAAAAAAAAAAAAQFHUiEDGKYK/Jqn4xkfnkkFHHB2T3aI2aVnsyfAleWj91+IR+chA4teKV8U9qhI9Vt9Yu/d70Z53DYezaX2zQdMwkriF0UQUq4iAgMYpgr8mqfjGR+eSQUccHZPdojZpWezJ8CV5aP3X4hH5xwAAAAGMAAAgAAAAIBkAACAAgAAgAEAAAAAAAAAIgIDi14pXxT2qEj1W31i793vRnncNh7NpfbNB0zCSuIXRRAc9X7GXTAAAIAAAACAZAAAgAIAAIABAAAAAAAAAAA=",
```

Comment:
<img width="1352" alt="Screenshot 2024-09-08 at 1 06 18 PM" src="https://github.com/user-attachments/assets/4a63b5ac-69a9-4433-90c5-00ce0e6a9643">
<img width="1338" alt="Screenshot 2024-09-08 at 1 06 42 PM" src="https://github.com/user-attachments/assets/e90f60b4-271a-4d3c-826e-8e2acac07ec1">

Reply:
What bip174.org does not reveal is the global xpubs!

Reply:
True! It also doesn't support psbt v2 :(. 

Something I've been thinking about for a while is it would be cool to do a "caravan-native" psbt explorer that was just using our code and plug some of these holes 😁